packages/text/src/Text.ts

```diff
@@ -713,6 +713,7 @@ export class Text extends Sprite
         }
         this._text = text;
         this.dirty = true;
+        this.updateText(true);
```

Comment:
I am a bit wary of this change, as I think it'll bring performance issues for fast changing text.

Text in PixiJS has always been 'lazy'. So if you updated text a million times within 1 frame..... it doesn't matter, the texture for text only got updated when it comes through to render.
Now we're saying, every time you change the text, it's going to update that texture immediately.

That's a different performance profile which may negatively effect content out there - I'm thinking of high scores or numbers counting up super fast.

Reply:
It is worth considering. Another option is to call `updateTransform` manually in `updateTexture`.